templates/patterns/navigation_list/pattern-navigation-list.html.twig

```diff
@@ -5,36 +5,51 @@
  */
 #}
 
-{% set _links = [] %}
-{% for link in links %}
-  {% set _link = {
-    link: {
-      label : link.label,
-      path : link.url,
-      is_external: link.is_external|default(false)
-    }
+{% if (title_url) %}
+  {% set title = {
+    label: title,
+    path: title_url,
+    icon_path: ecl_icon_path,
+    external: external_link|default(false)
   } %}
-  {% set _links = _links|merge([_link]) %}
-{% endfor %}
+{% endif %}
 
-{% set _secondary_links = [] %}
-{% for secondary_link in secondary_links %}
-  {% set _secondary_link = {
-    link: {
-      label : secondary_link.label,
-      path : secondary_link.url,
-      is_external: secondary_link.is_external|default(false)
-    }
-  } %}
-  {% set _secondary_links = _secondary_links|merge([_secondary_link]) %}
-{% endfor %}
+{% set _links = [] %}
+{% if links %}
+  {% set _primary_links = [] %}
+  {% for link in links %}
+    {% set _link = {
+      link: {
+        label : link.label,
+        path : link.url,
+        icon_path: ecl_icon_path,
+        external: link.is_external|default(false)
+      }
+    } %}
+    {% set _primary_links = _primary_links|merge([_link]) %}
+  {% endfor %}
+  {% set _links = _links|merge([_primary_links]) %}
+{% endif %}
+
+{% if secondary_links %}
+  {% set _secondary_links = [] %}
+  {% for secondary_link in secondary_links %}
+    {% set _secondary_link = {
+      link: {
+        label : secondary_link.label,
+        path : secondary_link.url,
+        icon_path: ecl_icon_path,
+        external: secondary_link.is_external|default(false)
```

Comment:
This is not adding the icon anymore.

On EWCMS:
<img width="547" alt="external secondary link" src="https://user-images.githubusercontent.com/22908988/182111560-47063ef7-dc65-4e17-a3de-39580bd939ce.png">

Reply:
Tested and have not found issue related to the pattern. Answered in the ticket.

Reply:
Indeed, my bad.